PurchasesTests/Misc/ISOPeriodFormatterTests.swift

```diff
@@ -17,73 +17,73 @@ class ISOPeriodFormatterTests: XCTestCase {
     
     func testStringFromProductSubscriptionPeriodDay() {
         if #available(iOS 11.2, macOS 10.13.2, tvOS 11.2, *) {
-            let formatter = Purchases.ISOPeriodFormatter()
+            let formatter = ISOPeriodFormatter()
             
             var period = SKProductSubscriptionPeriod(numberOfUnits: 1, unit: .day)
-            expect(formatter.string(from: period)) == "P1D"
+            expect(formatter.stringFromProductSubscriptionPeriod(period)) == "P1D"
```

Comment:
naming this `string(from productSubscriptionPeriod:)` would retain the old syntax both in objc and swift I believe

Reply:
this one you suggested: `string(fromProductSubscriptionPeriod period: SKProductSubscriptionPeriod)` is the one that works.

Reply:
ah, makes sense 👍